app/screens/post_options/post_options.tsx

```diff
@@ -63,17 +72,61 @@ const PostOptions = ({
     const shouldRenderFollow = !(sourceScreen !== Screens.CHANNEL || !thread);
     const shouldShowBindings = bindings.length > 0 && !isSystemPost;
 
-    const snapPoints = [
+    const snapPoints = useMemo(() => {
+        const items: BottomSheetProps['snapPoints'] = [1];
+        let count = shouldShowBindings ? 0.5 : 0;
+        if (canReply && sourceScreen !== Screens.THREAD) {
+            count++;
+        }
+
+        if (shouldRenderFollow) {
+            count++;
+        }
+
+        if (canMarkAsUnread && !isSystemPost) {
```

Comment:
```suggestion
        if (canMarkAsUnread) {
```
Looking in the index file, canMarkAsUnread already accounts for `isSystemPost`
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/7575921/210710567-4558e126-8a65-4fa1-b999-097b9028b746.png">